docs/user/openstack/README.md

```diff
@@ -176,68 +216,132 @@ Create and associate a floating IP to the ingress port:
 
-The installer should finish successfully, though it is still undergoing development and things might break from time to time.
+The installer should finish successfully, though it is still undergoing development and things might break from time to time. Look for a message like this to verify that your install succeded:
+
+```txt
+INFO Install complete!
+INFO To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/stack/ostest/auth/kubeconfig'
+INFO Access the OpenShift web-console here: https://console-openshift-console.apps.ostest.shiftstack.com
+INFO Login to the console with user: kubeadmin, password: xxx
+```
+
+## Checking Cluster Status
+
+If you want to see the status of the apps and services in your cluster during, or after a deployment, first export your admin kubeconfig:
+
+```sh
+export KUBECONFIG=ostest/auth/kubeconfig
+```
+
+After a finished deployment, there should be a node for each master and worker server created. You can check this whith the command:
+
+```sh
+oc get nodes
+```
+
+To se the version of your OCP cluster, do:
+
+```sh
+oc get cluster version
+```
+
+To see the status of you operators, do:
+
+```sh
+oc get clusteroperator
+```
+
+Finally, to see all the running pods in your cluster, you can do:
+
+```sh
+oc get pods -A
+```
+
+## Destroying The Cluster
+
+Destroying the cluster has been noticed to sometimes fail. We are working on patching this, but in a mean time the workaround is to simply run it again. To do so, point it to your cluster with this command:
+
+```sh
+./openshift-install --log-level debug destroy cluster --dir ostest
+```
+
+Then, you can delete the folder containing the cluster metatata:
+
+```sh
+rm -rf ostest/
+```
 
 ## Using an External Load Balancer
 
-This documents how to shift from the API VM load balancer, which is
-intended for initial cluster deployment and not highly available, to an
-external load balancer.
+This documents how to shift from the internal load balancer, which is intended for internal networking needs, to an external load balancer.
 
-The load balancer must serve ports 6443, 443, and 80 to any users of
-the system.  Port 22623 is for serving ignition startup configurations
-to the OpenShift nodes and should not be reachable outside of the cluster.
+The load balancer must serve ports 6443, 443, and 80 to any users of the system.  Port 22623 is for serving ignition start-up configurations to the OpenShift nodes and should not be reachable outside of the cluster.
 
 The first step is to add floating IPs to all the master nodes:
 
-* `openstack floating ip create --port master-port-0 <public network>`
-* `openstack floating ip create --port master-port-1 <public network>`
-* `openstack floating ip create --port master-port-2 <public network>`
+```sh
+openstack floating ip create --port master-port-0 <public network>
```

Comment:
It doesn't have to be public network. It has to be a network the external load balancer has access to though.